tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -37,55 +41,76 @@ bool IsRetriable(Status status) {
   }
 }
 
-Status CallWithRetries(const std::function<Status()>& f) {
-  int attempts = 0;
+void WaitBeforeRetry(const int64 delay_micros) {
+  const int64 random_micros = random::New64() % 1000000L;
```

Comment:
quick question should i be removing it from every place i have used? (like `kMaximumBackoffMicroseconds` , constructor args default value etc... )